Example/BasicExample/TwiliEngageExampleNotificationService/NotificationService.swift

```diff
@@ -39,7 +46,12 @@ class NotificationService: UNNotificationServiceExtension {
                 bestAttemptContent.attachments = [ mediaAttachment ]
             }
             
-            
+            var  notification = [String : Any]()
+            for (key, value) in bestAttemptContent.userInfo {
+                notification[key as! String] = value
+            }
+
+            Analytics.main.track(name: "Push Delivered", properties: notification)
```

Comment:
Im a bit concerned that we have to rely on the user to emit Push Delivered. Could the SDK do this?